packages/docs/src/components/integration-options/index.js

```diff
@@ -59,7 +59,17 @@ const options = [
         </p>
       </>
     ),
-  }
+  },
+  {
+    title: 'Option C: Portal API',
+    imageUrl: 'img/ReQ-01.png',
+    description: <>Fully managed solution, over REST API using Request Finance</>,
+    details: (
+      <>
+        <p>Reach out to us if you want to integrate Request Finance APIs.</p>
```

Comment:
Readded Portal API as an option for now.

Reply:
You can link to https://docs.request.finance if you want.
I will also link docs.request.network in docs.request.finance